packages/fx-core/src/plugins/resource/aad/plugin.ts

```diff
@@ -342,16 +345,13 @@ export class AadAppForTeamsImpl {
     return ResultFactory.Success();
   }
 
-  public async postProvisionUsingManifest(
-    ctx: PluginContext,
-    isLocalDebug = false
-  ): Promise<AadResult> {
+  public async postProvisionUsingManifest(ctx: PluginContext): Promise<AadResult> {
     TelemetryUtils.init(ctx);
     Utils.addLogAndTelemetryWithLocalDebug(
```

Comment:
By making this change, I think we can not get any post local debug telemetry?

Reply:
Thanks, fixed